lib/routes/account.js

```diff
@@ -178,9 +180,9 @@ module.exports = function (
             function (result) {
               account = result
 
-              log.activityEvent('account.created', request, {
+              log.activityEvent('account.created', request, metricsContext.add({
                 uid: account.uid.toString('hex')
-              })
+              }, form.metricsContext))
```

Comment:
Since `log.activityEvent` gets a reference to the request, could it do this context-adding transparently?  Or would that be too much of a layering violation?